src/mon/Monitor.cc

```diff
@@ -5983,6 +5999,19 @@ void Monitor::tick()
   }
 
   mgr_client.update_daemon_health(get_health_metrics());
+
+  // Memory Usage Logging: log current usage and a baseline
+  MemoryModel mm(g_ceph_context);
+  mm.sample();
+  dout(2) << "MON Memory usage: "
+	  << " total " << mm.last.get_total()
+	  << ", rss " << mm.last.get_rss()
+	  << ", heap " << mm.last.get_heap()
+	  << ", baseline " << baseline.get_heap()
```

Comment:
Please add `void MemoryModel::mem_snap_t::print(std::ostream&)` and make the printing consistent for all of these douts so you can:

`dout(2) << "MON memory usage: " << mm.last << dendl;`

and add units (i.e. `kiB` IIRC). The unit-less prints have been a source of confusion in the past for support engineers.